### PR TITLE
✏️ 1243 update warning wording

### PR DESCRIPTION
**Pull request by cy**
didn't catch that one sentence changed 😨

### CODE REVIEW

File: src/packages/@ncigdc/components/Modals/FirstTimeModal.js [hunk 1]
```diff
@@ -20,17 +20,18 @@ const FirstTimeModal = () => (
...
+
       <ExternalLink
         hasExternalIcon={false}
         href="https://docs.gdc.cancer.gov/Data/Release_Notes/Data_Release_Notes"
       >
         GDC Data Release Notes
-      </ExternalLink>
-      {" "}
-      prior to accessing this web site as the Release Notes provide details about data updates, known issues and workarounds.
+      </ExternalLink>&nbsp;
```
Comment (shanewilson) [inline src/packages/@ncigdc/components/Modals/FirstTimeModal.js]: do we want a `&nbsp;` here?

Comment (alex-wilmer) [inline src/packages/@ncigdc/components/Modals/FirstTimeModal.js]: both `{" "}` and `&nbsp;` make sense to me in this context

Comment (shanewilson) [inline src/packages/@ncigdc/components/Modals/FirstTimeModal.js]: `&nbsp;` is different though isn't it? the line won't wrap around it. If it is just used to get around prettier I don't think we should use it.

Comment (alex-wilmer) [inline src/packages/@ncigdc/components/Modals/FirstTimeModal.js]: @cy is that why you changed it? because prettier weirdness?

Comment (cy) [inline src/packages/@ncigdc/components/Modals/FirstTimeModal.js]: yeah

Comment (cy) [inline src/packages/@ncigdc/components/Modals/FirstTimeModal.js]: I didn't like the extra newline just for {" "} but I can change it back :woman_shrugging:

Comment (shanewilson) [inline src/packages/@ncigdc/components/Modals/FirstTimeModal.js]: I think messing with prettier output sends us down a dangerous path of bike shedding.

File: src/packages/@ncigdc/components/Modals/FirstTimeModal.js [hunk 2]
```diff
@@ -29,8 +29,9 @@ const FirstTimeModal = () => (
         GDC Data Release Notes
       </ExternalLink>
       {" "}
-      prior to accessing this web site as the Release Notes provide details about data updates, known issues and workarounds.
+      prior to accessing this web site as the Release Notes provide details about data updates, and known issues and workarounds.
```
Comment (shanewilson) [inline src/packages/@ncigdc/components/Modals/FirstTimeModal.js]: I don't think you need this comma since there are only two things?

Comment (alex-wilmer) [inline src/packages/@ncigdc/components/Modals/FirstTimeModal.js]: oh this came back? I remember asking you to remove that `and`.. did someone request this to come back?

Comment (shanewilson) [inline src/packages/@ncigdc/components/Modals/FirstTimeModal.js]: or if `known issues and workarounds` are two things I don't think you need the extra `and`

Comment (alex-wilmer) [inline src/packages/@ncigdc/components/Modals/FirstTimeModal.js]: I think it should read... "details about data updates, known issues and workarounds"

Comment (alex-wilmer) [inline src/packages/@ncigdc/components/Modals/FirstTimeModal.js]: left it is there

<-----------

Comment (cy) [inline src/packages/@ncigdc/components/Modals/FirstTimeModal.js]: where is this arrow pointing

Comment (cy) [inline src/packages/@ncigdc/components/Modals/FirstTimeModal.js]: oh I see the extra 'and', didn't read it carefully, copy and pasted it from the ticket, which was a huge text blob. From the diff, looks like the change is adding this sentence in the warning `Data in the GDC is considered provisional as the GDC applies state-of-the art analysis pipelines which evolve over time.`

File: src/packages/@ncigdc/components/Modals/FirstTimeModal.js [hunk 3]
```diff
@@ -40,8 +40,7 @@ const FirstTimeModal = () =>
         GDC Data Release Notes
       </ExternalLink>
       {' '}
-      prior to accessing this web site as the Release Notes provide details
-      about data updates, known issues and workarounds.
+      prior to accessing this web site as the Release Notes provide details about data updates, and known issues and workarounds.
```
Comment (shanewilson) [inline src/packages/@ncigdc/components/Modals/FirstTimeModal.js]: ~and~

File: src/packages/@ncigdc/components/Modals/FirstTimeModal.js (hunk 2 above)
Comment (alex-wilmer) [inline src/packages/@ncigdc/components/Modals/FirstTimeModal.js]: oh haha the side by side view.. i'm pointing to the way it was before. Did someone request to add this "and" back? I don't think it reads well

File: src/packages/@ncigdc/components/Modals/FirstTimeModal.js [hunk 4]
```diff
@@ -31,7 +31,9 @@ const FirstTimeModal = () =>
     </p>
     <p>
       <b>WARNING</b>
-      : Please read the
+      : Data in the GDC is considered provisional as the GDC applies
+      state-of-the art analysis pipelines which evolve over time. Please read
+      the
```
Comment (cy) [inline src/packages/@ncigdc/components/Modals/FirstTimeModal.js]: the on one line, thanks prettier 🚲

**Pull request merged**